themes/classic/_dev/js/theme.js

```diff
@@ -8,17 +8,24 @@ import '../css/theme';
 import './setup-rivets';
 import './checkout';
 import './facets';
-import './product';
+
+import DropDown from './components/drop-down';
+import TopMenu from './components/top-menu';
 
 import prestashop from 'prestashop';
 import EventEmitter from 'events';
-import {psShowHide} from './common';
+import {
+  psShowHide
+}
+from './common';
 
 // "inherit" EventEmitter
 for (var i in EventEmitter.prototype) {
-    prestashop[i] = EventEmitter.prototype[i];
+  prestashop[i] = EventEmitter.prototype[i];
 }
 
 $(document).ready(() => {
+  let dropDown = new DropDown();
+  let topMenu = new TopMenu();
```

Comment:
As a general rule, [constructors should not have side-effects](http://blog.millermedeiros.com/constructors-should-not-cause-side-effects/).

I think moving the contents of both constructors into methods called something like `bindToDOM` or `setup` would be cleaner.

If you chose to put this in the constructor to save a line, the following is valid JS syntax:

```javascript
new DropDown().bindToDOM();
```